pom.xml

```diff
@@ -42,6 +42,12 @@
             <artifactId>credentials</artifactId>
             <version>1.21</version>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
```

Comment:
:ant: You do not need to specify a `junit` test dependency, it is automatic for all plugins.